### ISSUE TITLE
disable weather

### ISSUE THREAD

**Issue by pkarza** [Qrome/marquee-scroller#70]
Would it be possible to load this without weather data?  The news works great, but as you know I am having issues with weather pulls - maybe because of my ISP or being in Canada... I don't know.

**Comment by Qrome** [comment]
No, it uses the city ID to get the latitude and longitude for the time zone.   It currently requires the weather.

**Comment by Qrome** [comment]
The original purpose of the scroller was for weather -- I do see putting it in as an option to turn on or off to be nice.   However, the TimeZone option and DST is still a major headache for the user.   One of my development goals is to make it as simple for the user as possible.   TimeZone and DST for international solutions are a huge issue.

**Comment by wabbitguy** [comment]
Although it would be impossible to make it foolproof, I find DST the easiest part because the timezone/dst is simply set once in the code, then you upload the code and it's done forever. No reason to complicate it beyond that.

Bad enough I have to go round to every appliance in the house to reset the time without having to adjust all the 8266's...Plus there are a lot of people who work weekends and need the correct time...

I'm going to see if I can convert to a regular NTP scheme with DST rules and then possibly enhance the weather to get a future forecast (more important to me than current conditions). Maybe even see if I can update to the Parola library to remove the 16 matrix limit...lotsa fun...:-)
